Sparkle/SUUpdateAlert.m

```diff
@@ -107,15 +107,10 @@ - (IBAction)remindMeLater:(id)__unused sender
 
 - (void)displayReleaseNotes
 {
-    // Set the default font
     [self.releaseNotesView setPreferencesIdentifier:SUBundleIdentifier];
     WebPreferences *prefs = [self.releaseNotesView preferences];
-    NSString *familyName = [[NSFont systemFontOfSize:8] familyName];
-    if ([familyName hasPrefix:@"."]) { // 10.9 returns ".Lucida Grande UI", which isn't a valid name for the WebView
-        familyName = @"Lucida Grande";
-    }
-    [prefs setStandardFontFamily:familyName];
-    [prefs setDefaultFontSize:(int)[NSFont systemFontSizeForControlSize:NSSmallControlSize]];
+    [prefs setStandardFontFamily:@"-apple-system-font"];
```

Comment:
You are right `-apple-system` is the new recommended token. But it was added recently (https://bugs.webkit.org/show_bug.cgi?id=144484), so I believe we can’t use it in older systems.